components/search/DateFilter.vue

```diff
@@ -1,79 +1,78 @@
 <template>
```

Comment:
Changes to this component's template are just to wrap it in a `b-form-group` (rendered as a `fieldset`) and set its heading with the `label` property (rendered as a `legend`) instead of in a `strong`.